internal/api/integrations/k8s/k8s.go

```diff
@@ -58,11 +64,14 @@ func (k *K8sClient) ListAll(namespace string) ([]models.Release, error) {
 	for _, r := range releaseList {
 		annotations := r.GetAnnotations()
 		autoDeploy, err := strconv.ParseBool(annotations[annotationFor("autodeploy")])
-		if err != nil {
-			return nil, err
-		}
+
+		k.logger.Log(err)
```

Comment:
This error needs more context. Seeing `unable to parse "ture" as bool` (or something like that) in the log output of this service won't be useful or actionable. We should use [`errors.Wrap`](https://godoc.org/github.com/pkg/errors#Wrapf) to provide more detail, like what caused the parse failure (parsing autodeploy annotation) and which release contains the invalid annotation

We should also only log the error if it's `!= nil`. The logger also needs 2 arguments, a key-value pair. For simple error handling like this, we usually use `Log("error", err)`